tdd-todo-list.CSharp.Main/Todo.cs

```diff
@@ -0,0 +1,118 @@
+﻿namespace tdd_todo_list.CSharp.Main
+{
+    public class Todo
+    {
+        public int Id { get; }
+        public string Description { get; set; }
+        public bool Done { get; set; }
+        public DateTime CreatedAt { get; }
+
+        // Construct that gets called when creating a Todo
+        public Todo(string taskDesc, int taskId)
```

Comment:
It would be better to autogenerate the ID to ensure uniqueness, rather than trusting the caller of the constructor to provide a sensible ID